monkey/agent_plugins/exploiters/smb/src/smb_options.py

```diff
@@ -0,0 +1,28 @@
+from pydantic import Field
+
+from common.base_models import InfectionMonkeyBaseModel
+
+
+class SMBOptions(InfectionMonkeyBaseModel):
+    agent_binary_upload_timeout: float = Field(
+        default=30.0,
+        gt=0.0,
+        le=100.0,
+        description="The timeout (in seconds) for uploading the Agent binary"
+        " to the target machine.",
+    )
+    use_kerberos: bool = Field(
+        default=False, description="Use Kerberos authentication for RPC transport."
+    )
+    rpc_connect_timeout: float = Field(
+        default=15.0,
+        gt=0.0,
+        le=100.0,
+        description="The maximum time (in seconds) to wait for a response on an RPC connection.",
+    )
+    smb_connect_timeout: float = Field(
+        default=15.0,
+        gt=0.0,
+        le=100.0,
+        description="The maximum time (in seconds) to wait for a response on an SMB connection.",
+    )
```

Comment:
Should these just be the same option?

Reply:
https://github.com/guardicore/monkey/pull/3089#discussion_r1133563064 Yep.